Documentation/concepts/kubernetes/compatibility-table.rst

```diff
@@ -2,48 +2,6 @@
 | Cilium          | CNP and CCNP   |
 | Version         | Schema Version |
 +-----------------+----------------+
-| v1.7.0-rc1      | 1.15           |
-+-----------------+----------------+
-| v1.7.0-rc2      | 1.16           |
-+-----------------+----------------+
-| v1.7.0-rc3      | 1.16           |
-+-----------------+----------------+
-| v1.7.0-rc4      | 1.16           |
-+-----------------+----------------+
-| v1.7.0          | 1.16           |
-+-----------------+----------------+
-| v1.7.1          | 1.16           |
-+-----------------+----------------+
-| v1.7.2          | 1.17           |
-+-----------------+----------------+
-| v1.7.3          | 1.17           |
-+-----------------+----------------+
-| v1.7.4          | 1.17           |
-+-----------------+----------------+
-| v1.7.5          | 1.17           |
-+-----------------+----------------+
-| v1.7.6          | 1.18           |
-+-----------------+----------------+
-| v1.7.7          | 1.18.1         |
-+-----------------+----------------+
-| v1.7.8          | 1.18.1         |
-+-----------------+----------------+
-| v1.7.9          | 1.18.1         |
-+-----------------+----------------+
-| v1.7.10         | 1.18.1         |
-+-----------------+----------------+
-| v1.7.11         | 1.18.1         |
-+-----------------+----------------+
-| v1.7.12         | 1.18.1         |
-+-----------------+----------------+
-| v1.7.13         | 1.18.1         |
-+-----------------+----------------+
-| v1.7.14         | 1.18.1         |
-+-----------------+----------------+
-| v1.7.15         | 1.18.1         |
-+-----------------+----------------+
-| v1.7            | 1.18.1         |
```

Comment:
Just checking, were the `v1.7*` entries removed automatically when running `./Documentation/check-crd-compat-table.sh v1.10`? Asking because I ran it and they were still here for me, but then I haven't followed the previous steps for the feature release process, just tried that command locally. In the case you had to remove then manually, should [the documentation](https://docs.cilium.io/en/latest/contributing/release/feature/) be updated?

Reply:
Oh if I understand correctly this is in progress, in that commit https://github.com/cilium/cilium/pull/15869/commits/73a3b8bd8cc9b72f5a50a57503e94963e8b402a8. Ok then :+1:.